docs/ReleaseFlow.md

```diff
@@ -22,3 +22,20 @@ $ git push origin prepare-release-v3.0.2
 ```
 
 * In this case, the `Github Actions` for the release will be executed upon `Git push`.
+
+* If you release the most recent version of charts, you need to update the version of the main branch as well.
```

Comment:
Can you clarify what needs to be updated?
Also just wondering if it can be done automatically?

Reply:
> Can you clarify what needs to be updated?

I added the descriptions about what we need to update in 3059edffc83db53459990a30dc77d14f67a9838b.

However, I think these steps are almost **release steps** (they are very similar to release steps).
So, I think that it is better to update entire docs to explain release steps in more detail.

> Also just wondering if it can be done automatically?

I think there is no way to automatically update now.

As manually ways, I think there are two options as follows.
* Create PR to keep main branch latest after releasing latest charts.
* Trigger release workflow by merging into the main, only when we wil release latest version charts.

For update the docs that as mentioned above, I would like to discuss about release flow.
Can I talk to you about this PR in the next week?